packages/api-headless-cms/src/content/plugins/schema/resolvers/manage/resolveGet.ts

```diff
@@ -41,7 +41,7 @@ const getValuesFromArgs = (args?: ValuesFromArgsParams): ArgsValues => {
             }
         );
     }
-    if (type && possibleTypes.includes(type) === false) {
+    if (status && possibleTypes.includes(status) === false) {
         throw new WebinyError(
             `Status cannot be anything other than ${possibleTypes.join(", ")}.`,
```

Comment:
```
Status can only be one of the following values:  ${possibleTypes.join(" | ")}.
```


Btw. can we define `status` as an `enum`  so GraphQL takes care of the validation for us?